### PR TITLE
data:HarshitMahara

### PR DESCRIPTION
**Pull request by HarshitMahara**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5855"></a>

### CODE REVIEW

File: data/HarshitMahara.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "GitHub",
+      "url": "https://github.com/HarshitMahara",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/harshit-singh-mahara-1084b9227/",
+      "icon": "FaLinkedin"
+    },
```
Comment (khushalarora) [inline data/HarshitMahara.json:25]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @HarshitMahara, thank you for contributing  :tada:

File: data/HarshitMahara.json (hunk 1 above)
Comment (HarshitMahara) [reply data/HarshitMahara.json:25]: okay

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.eddiehub.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**